docs/source/deployment/airflow.md

```diff
@@ -0,0 +1,181 @@
+# Apache Airflow
+
+Apache Airflow is a popular open-source workflow management platform. It is a suitable engine to orchestrate and execute a pipeline authored with Kedro because workflows in Airflow are modelled and organised as [DAGs](https://en.wikipedia.org/wiki/Directed_acyclic_graph).
+
+## How to run a Kedro pipeline on Apache Airflow with Astronomer
+
+The following tutorial shows how to deploy an example [Spaceflights Kedro project](https://docs.kedro.org/en/stable/tutorial/spaceflights_tutorial.html) on [Apache Airflow](https://airflow.apache.org/) with [Astro CLI](https://docs.astronomer.io/astro/cli/overview), a command-line tool created by [Astronomer](https://www.astronomer.io/) that streamlines the creation of local Airflow projects. We will guide you through deploying it locally first, and then transitioning to Astro Cloud.
+
+[Astronomer](https://docs.astronomer.io/astro/install-cli) is a managed Airflow platform which allows users to spin up and run an Airflow cluster easily in production. Additionally, it also provides a set of tools to help users get started with Airflow locally in the easiest way possible.
+
+### Strategy
+
+The general strategy to deploy a Kedro pipeline on Apache Airflow is to run every Kedro node as an [Airflow task](https://airflow.apache.org/docs/apache-airflow/stable/concepts/tasks.html) while the whole pipeline is converted into a [DAG](https://airflow.apache.org/docs/apache-airflow/stable/concepts/dags.html) for orchestration purpose. This approach mirrors the principles of [running Kedro in a distributed environment](distributed.md).
+
+Each node will be executed within a new Kedro session, which implies that `MemoryDataset`s cannot serve as storage for the intermediate results of nodes. Instead, all datasets must be registered in the [`DataCatalog`](https://docs.kedro.org/en/stable/data/index.html) and stored in persistent storage. This approach enables nodes to access the results from preceding nodes.
+
```

Comment:
We haven't done a release of `kedro-airflow` with https://github.com/kedro-org/kedro-plugins/pull/241 yet, but this would not necessarily be required after that 🤔

Reply:
Thank you for pointing that out, @ankatiyar ; I overlooked it. Indeed, it seems that after the release, there will be two options available: grouping nodes with two commands, or writing MemoryDatasets to the DataCatalog!

Reply:
I added a section about the grouping strategy following the kedro-airflow 0.9.0 release.